ProjectForce/force-app/main/default/aura/newD3Comp/newD3CompController.js

```diff
@@ -196,27 +207,40 @@ myAction : function(component, event, helper) {
                         labelString: 'Month',
                     },
                     gridLines: {
-                    }, 
+                        display:false,
+                    },
                     stacked: true
                 }],
                 yAxes: [{
                     gridLines: {
-                        display:false,
-                        color: "#fff",
-                        zeroLineColor: "#fff",
-                        zeroLineWidth: 0
+                        display:true,
+                        offsetGridLines:true,
+                        zeroLineWidth: 1,
+                        color: 'black',
+                        drawTicks: false
                     },
                     ticks: {
                         fontFamily: "'Futura', sans-serif",
-                        fontSize:11,
-                        display: false
-                        
-                        
+                        fontSize:20,
+                        display: true,
+                        callback: function(value, index, values) {
+                            var tra = JSON.parse(localStorage.getItem("tracks"));// getting values from helper
```

Comment:
Please put inline comments above the thing it's talking about. It's the convention followed elsewhere